ArduPlane/config.h

```diff
@@ -340,9 +340,14 @@
 #define PARACHUTE ENABLED
 #endif
 
+//////////////////////////////////////////////////////////////////////////////
+// Payload Gripper
+#ifndef GRIPPER_ENABLED
```

Comment:
I think it's also probably worth a check of `#if !HAL_MINIMIZE_FEATURES` as well before we enable it. The gripper is probably not something that is ever needed on older boards, and is a feature they haven't had before. (It's a fairly small feature, only 1120 bytes, but I think it's still worth keeping out of the low resource builds)